### DISCUSSION TITLE
Missing `aptos` command

### DISCUSSION THREAD

**Discussion by Alexzander-Stone** [aptos-labs/aptos-developer-discussions#84] [Questions] [answered]
Asked by `Marshy` on Discord (#dev-discussions):
>  The term 'aptos' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
> the spelling of the name, or if a path was included, verify that the path is correct and try again.
> At line:1 char:1
> \+ aptos move test
> \+ \~
>     \+ CategoryInfo          : ObjectNotFound: (aptos:String) [], CommandNotFoundException
>     \+ FullyQualifiedErrorId : CommandNotFoundException
>
> please how can i fix this?

**Keywords**
aptos, command-line, CLI, installation, setup

**Comment by Alexzander-Stone** [comment, marked as answer]
The `aptos` command isn't available in the terminal that you're working with. 
Typically, this means the [Aptos CLI installation directions](https://aptos.dev/tools/aptos-cli/install-cli/) were followed incorrectly.

Can you detail the installation process you took? As well as your operating system and chosen terminal?

**Reply by gregnazario** [reply]
Looks like Windows, PowerShell.  Please follow the Windows instructions provided above
